src/domain/relay/contracts/__tests__/proxy-factory-decoder.helper.spec.ts

```diff
@@ -7,18 +7,23 @@ describe('ProxyFactoryDecoder', () => {
   let target: ProxyFactoryDecoder;
 
   beforeEach(() => {
-    jest.clearAllMocks();
+    jest.resetAllMocks();
     target = new ProxyFactoryDecoder();
   });
 
   it('decodes a createProxyWithNonce function call correctly', () => {
     const createProxyWithNonce = createProxyWithNonceEncoder();
-    const args = createProxyWithNonce.build();
     const data = createProxyWithNonce.encode();
 
+    const expectedArgs = createProxyWithNonce.build();
+
```

Comment:
nitpick:
```suggestion
```

Reply:
Removed in 2b75f622363761d6856fe5e53d70bcc1d0988005.